File: saida.py
def ligar(id_device):
    print(f"{id_device} ligado!")

def desligar(id_device):
    print(f"{id_device} desligado!")

def alerta(id_device, msg):
    print(f"{id_device} recebeu o alerta:\n{msg}")

def alerta_var(id_device, msg, var):
    print(f"{id_device} recebeu o alerta:\n{msg} {var}")

dispositivos = ['Lampada']

potencia = 10

# Código traduzido:

potencia = 10
if (potencia > 5):
    ligar("Lampada")

File: saida.c
#include <stdio.h>
#include <string.h>

void ligar(char* id) {
    printf("%s ligado!\n", id);
}

void desligar(char* id) {
    printf("%s desligado!\n", id);
}

void alerta(char* id, char* msg) {
    printf("%s recebeu o alerta: %s\n", id, msg);
}

void alerta_var(char* id, char* msg, char* var) {
    printf("%s recebeu o alerta: %s %s\n", id, msg, var);
}
int main() {
    alerta_var("Termometro", "Temperatura esta em", temperatura);
    return 0;
}
